fake_switches/juniper/juniper_netconf_datastore.py

```diff
@@ -307,6 +311,17 @@ def parse_vlans(self, conf, etree_conf):
 
         return handled_elements
 
+    def parse_native_vlan(self, interface_node, port):
+        if len(interface_node.xpath("unit/family/ethernet-switching/native-vlan-id")) == 1 and interface_node.xpath("unit/family/ethernet-switching/native-vlan-id")[0].text is not None:
```

Comment:
2 times the same xpath on the same line, you could put 

interface_node.xpath("unit/family/ethernet-switching/native-vlan-id")

in a variable